packages/@icon-magic/icon-models/src/asset.ts

```diff
@@ -49,7 +51,7 @@ export class Asset {
    * If an absolute path is passed in, store the relative path relative to the
    * iconPath
    */
-  set path(filePath: string) {
+  setPath(filePath: string) {
```

Comment:
When is this called? We could get rid of it entirely when we remove `unresolvedPath`.

Reply:
It's not called so it can be removed but `unresolvedPath` is being used, see below comment.